components/ContactForm.tsx

```diff
@@ -1,31 +1,34 @@
-import React from "react";
-import {Color} from "../Color.enum";
-import {Box, Text, Button} from "rebass";
+import React from 'react';
+import {Color} from '../Color.enum';
+import {Box, Text, Button} from 'rebass';
 import {Input, Textarea} from '@rebass/forms';
-import {theme} from "../theme";
-import {CONSTANTS} from "../shared/constants";
+import {theme} from '../theme';
+import {CONSTANTS} from '../shared/constants';
 
 type TContactFormProps = {
     color: Color;
 };
 
 export const ContactForm: React.FC<TContactFormProps> = ({children, color}) => {
-    // write some code
 
-    const [content, setContent] = React.useState('default');
+    const [content, setContent] = React.useState('');
```

Comment:
Consider to rename `content` to something like `textAreaContent`.

```suggestion
    const [textAreaContent, setTextAreaContent] = React.useState('');
```